src/main/kotlin/org/rust/ide/structure/RustConstTreeElement.kt

```diff
@@ -6,7 +6,7 @@ import org.rust.lang.core.psi.RustConstItemElement
 
 class RustConstTreeElement(element: RustConstItemElement) : PsiTreeElementBase<RustConstItemElement>(element) {
 
-    override fun getPresentableText(): String? = element?.name
+    override fun getPresentableText() = "${element?.name}: ${element?.type?.text}"
```

Comment:
If we don't have type (is this possible?), then we will be left with ugly trailing colon